pkg/internal/apis/certmanager/types.go

```diff
@@ -159,27 +159,27 @@ const (
 type KeyUsage string
 
 const (
-	UsageSigning            KeyUsage = "signing"
-	UsageDigitalSignature   KeyUsage = "digital signature"
-	UsageContentCommittment KeyUsage = "content commitment"
```

Comment:
Ouch. I don't think this can be changed unfortunately. We are in [good company](https://en.wikipedia.org/wiki/HTTP_referer) though

Reply:
Can I convince you to accept the action to prevent future instances? :-)

Reply:
One possible solution would be to add the correct spelling to the map, which would not create any backwards compatibility issues. 

Re: adding the action I think we would want to add something similar to our pull-cert-manager-verify job rather than having another action, similar to how we spell-check the cert-manager website. But I'm not opposed to just adding the action. I'll ask other maintainers 🙂

Reply:
> HTTP referer

Wow, I think I always misspelled that due to the HTTP misspelling 😅

There is a "référer" word in French, looks like the misspell comes from a french influence!

Reply:
It's nice though, typing "referer" on Google immediately gives you access to the HTTP documentation 😅

Reply:
@jakexks: if you're open to doing an add instead of a swap, I'd be happy to do that (in a distinct PR since it's a different duck and would want different review).

Reply:
I propose that we add a fun easter egg comment on top of `UsageContentCommitment`:

```go
	// This misspelling will live forever in our hearts and will remind us
	// of the Unix dictionary that happened to not know how to spell
	// "referrer" in 1992.
	UsageContentCommittment KeyUsage = "content commitment"
```

Reply:
As long as the comment begins with Deprecated: (for tripping linters / special ections in generated documentation) it can be anything 🙂  e.g. https://golang.org/src/net/dial.go?h=Deprecated%3A#L59

Reply:
Aaaahhh yes, I forgot about the generated documentation... :-(